src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    public Truck(String nameMachine) {
+        super(nameMachine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(getNameMachine() + " started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(getNameMachine() + " stoped working");
```

Comment:
There is a typo in the message 'stoped'. The correct spelling is 'stopped'. Also, the message should be more informative and indicate that a 'Truck' is the type of machine that stopped working. For example, 'Truck <name> stopped working.'